src/onlineHelp/forkliftHelp/topics/KeyTerminologyHelpTopic.tsx

```diff
@@ -0,0 +1,271 @@
+            ),
+          },
+        ],
+        title: t('Kubernetes'),
+      },
+      {
+        id: 'terminology-node',
+        subListStyleType: ListStyleType.DESCRIPTIONS,
+        subTopics: [
+          {
+            id: 'node-definition',
+            title: t('A virtual or bare-metal machine in a Kubernetes cluster.'),
+          },
+        ],
+        title: t('Node'),
+      },
+      {
+        id: 'terminology-migration-types',
+        subListStyleType: ListStyleType.DESCRIPTIONS,
+        subTopics: [
+          {
+            id: 'migration-types-definition',
+            title: t('A cold migration moves a shut-down virtual machine between hosts.'),
+          },
+        ],
+        title: t('Migration types (cold vs warm vs)'),
+      },
+      {
+        id: 'terminology-network-map',
+        subListStyleType: ListStyleType.DESCRIPTIONS,
+        subTopics: [
+          {
+            id: 'network-map-definition',
+            title: t(
+              'A configuration that defines how the networks from your source platform (VMware vSphere, Red Hat Virtualization, OpenStack, or OVA files) will connect to the networks in OpenShift Virtualization.',
+            ),
+          },
+        ],
+        title: t('Network map'),
+      },
+      {
+        id: 'terminology-persistent-volume',
+        subListStyleType: ListStyleType.DESCRIPTIONS,
+        subTopics: [
+          {
+            id: 'persistent-volume-definition',
+            title: t(
+              'Provides a storage framework to allow administrators to provision persistent storage for a cluster.',
+            ),
+          },
+        ],
+        title: t('Persistent Volume (PV)'),
+      },
+      {
+        id: 'terminology-persistent-volume-claim',
+        subListStyleType: ListStyleType.DESCRIPTIONS,
+        subTopics: [
+          {
+            id: 'persistent-volume-claim-definition',
+            title: t(
+              'Requests persistent volume (PV) resources without having specific knowledge of the underlying storage infrastructure.',
+            ),
+          },
+        ],
+        title: t('Persistent Volume Claim (PVC)'),
+      },
+      {
+        id: 'terminology-project',
+        subListStyleType: ListStyleType.DESCRIPTIONS,
+        subTopics: [
+          {
+            id: 'project-definition',
+            subListStyleType: ListStyleType.SQUARE,
+            subTopics: [
+              {
+                id: 'project-definition-plan-project',
+                title: (
+                  <>
+                    <strong>{t('Plan project')}</strong>
+                    <br />
+                    {t(
+                      'The project, within your selected target provider, in which your virtual machines will be migrated.',
+                    )}
+                  </>
```

Comment:
This pattern is often repeated in all topics, maybe creating a small helper component would make this file (and other topic files) a bit more clean, what do you think?

Reply:
Also for cases like this where you need to translate text with components inside like strong, br, external link etc.
you can use ForkliftTrans component to wrap it and omit the t function and get the translation in a bit cleaner fashion, something like this:
```
<ForkliftTrans>
  <strong>Target project</strong>
  <br />
  The project, within your selected target provider, in which your virtual machines will be migrated.
</ForkliftTrans>
```